packages/webb-ui-components/postcss.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  plugins: {
+    tailwindcss: {},
+    autoprefixer: {},
+  },
+};
```

Comment:
Can we consider inheriting the setting from the root config file? It'll be more accessible when we have any updates on the config, we won't have to change in multiple places.

Reply:
Sth likes this works on my machine.

```js
const config = require('../../postcss.config');

module.exports = config;
```